test/fulfillmentSpec.js

```diff
@@ -332,162 +326,4 @@ describe('PUT /fulfillment', function () {
     expect((yield getAccount('alice')).balance).to.equal(100)
     expect((yield getAccount('bob')).balance).to.equal(0)
   })
-
-  it('should trigger subscriptions when notification is executed', function * () {
```

Comment:
Done: moved relevant tests to `notificationSpec.js` and added new tests to check the fulfillment. See https://github.com/interledger/five-bells-ledger/blob/ws_notifications/test/notificationSpec.js#L344